internal/controllers/objectdeployments/objectdeployment_controller_test.go

```diff
@@ -0,0 +1,179 @@
+package objectdeployments
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/errors"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	corev1alpha1 "package-operator.run/apis/core/v1alpha1"
+	"package-operator.run/internal/testutil"
+)
+
```

Comment:
I think it would be safer to create a separate test scheme and not use the scheme of another test.
```suggestion
var testScheme = runtime.NewScheme()
```